### ISSUE TITLE
Telegram reporter

### ISSUE THREAD

**Issue by ivanhercaz** [mimiquate/tower#70]
Hi!

**TLDR**: there is available a Telegram reporter, [TowerTelegram](https://github.com/ivanhercaz/tower_telegram).

@grzuy mentioned me this project in an issue I reported in [ErrorTracker](https://github.com/elixir-error-tracker/error-tracker) (elixir-error-tracker/error-tracker#54). After the comment I think "wow!", because both tools seems to work really nice by itself but both joined must be like a supertool for error tracking and reporting! So, first of all, thank you very much to mimiquate team and ErrorTracker team for these awesome tools.

Then I was thinking in going to sleep but the curiosity kills the cat and I started reading `tower` code and the reporters code, and then I see the [TowerEmail](https://github.com/mimiquate/tower_email), and I think "oh, curious, it is a powerful but easy way to create a report". And well, I couldn't go to rest until develop a bit a Telegram reporter.

Telegram is one of my most used tools for communicating and it is used by many developer mates, so I think it can be very useful. So I started to study the Tower reporters and develop a bit of the idea. I think it can be done better and maybe include more configuration options, and of course, it isn't well tested nor have tests, but here it is, [TowerTelegram](https://github.com/ivanhercaz/tower_telegram), in case you want to take a look or suggest something.

And well, I think I become delirious and talk a lot, probably more than the necessary to share with you the idea, so I just wanted to share one last thing:

![Captura de pantalla_2024-10-05_02-25-06](https://github.com/user-attachments/assets/b2b5e9db-26e3-4ac1-8f78-9a977986e1fb)

One more time, thank you for all your work with this project. And excuse me for maybe talking/writing more than the necessary 😅 .

Regards

**Comment by grzuy** [comment]
Hi @ivanhercaz,

So great to read that you got it working so quickly! :tada: 
Glad to start seeing other Tower reporters written by others appear and be useful :clap: 

Thanks a lot for sharing that!

**Comment by ivanhercaz** [comment]
Once I will make some changes a document it a bit better, I will release the package on Hex and notice in this issue.

Of course, consider closing this issue unless you want to keep it open for some reason. If some of you have any idea or suggestion for the Telegram reporter, please, feel free to open an issue there 🥰.

**Comment by grzuy** [comment]
For the record, leaving the link to your repo (given it didn't get linked on your initial comment), for the benefit of anyone else looking for a similar solution: https://github.com/ivanhercaz/tower_telegram.

**Comment by ivanhercaz** [comment]
Ups, I though I linked the repository in some part of the message but yes, it wasn't linked. Now I added the link and a **TLDR** message in the top.

Thank you, @grzuy!
